core/modules/admin_bar/js/admin_bar.js

```diff
@@ -418,6 +418,7 @@ Backdrop.adminBar.behaviors.search = function (context, settings, $adminBar) {
     else {
       $(this).removeClass('active-search-item');
     }
+    $adminBar.find('[class=""]').removeAttr('class');
```

Comment:
Instead of searching the entire admin bar, we could just check the affected item after a class is removed. Something like this:

```
$this.removeClass('active-search-item');
if ($this.is('[class=""]')) {
  $this.removeAttr('class');
}
```

Reply:
Tried that exact cde, but it didn't work. I think because there's another place just above that removes that class as well. See updated PR for my solution that seems to work, and hopefully isn't too bad re. performance.